scripts/alternate-server-replay-4.0.4.py

```diff
@@ -0,0 +1,130 @@
+# This file was copied from  mitmproxy/mitmproxy/addons/serverplayback.py release tag 4.0.4
```

Comment:
As we're only supporting one version of mitmproxy, could we rename this file to remove the version number?